ipywidgets/static/widgets/js/manager.js

```diff
@@ -461,54 +335,248 @@ define([
                             // Request the state from the backend
                             return model.request_state();
                         });
-                    } else { // dead comm
-                        return kernel.widget_manager.new_model({
-                            model_id: model_id,
-                            model_name: state[model_id].model_name,
-                            model_module: state[model_id].model_module,
-                        }).then(function(model) {
-                            return model._deserialize_state(state[model_id].state).then(function(state) {
-                                model.set_state(state);
-                                return model;
-                            });
+                    });
+                } else { // dead comm
+                    return that.new_model({
+                        model_id: model_id,
+                        model_name: state[model_id].model_name,
+                        model_module: state[model_id].model_module,
+                    }).then(function(model) {
+                        return model._deserialize_state(state[model_id].state).then(function(state) {
+                            model.set_state(state);
+                            return model;
                         });
-                    }
-                }));
-            });
+                    });
+                }
+            }));
+        });
 
-            // Display all the views
-            return all_models.then(function(models) {
-                return Promise.all(_.map(models, function(model) {
-                    // Display the views of the model.
-                    return Promise.all(_.map(state[model.id].views, function(cell_index) {
-                        var cell = that.notebook.get_cell(cell_index);
-                        return that.display_view_in_cell(cell, model);
-                    }));
+        // Display all the views
+        return all_models.then(function(models) {
+            return Promise.all(_.map(models, function(model) {
+                // Display the views of the model.
+                return Promise.all(_.map(state[model.id].views, function(options) {
+                    return that.display_model(undefined, model, options);
                 }));
-            });
+            }));
         }).catch(utils.reject('Could not set widget manager state.', true));
     };
 
-    WidgetManager.prototype._get_connected_kernel = function() {
+    ManagerBase.prototype._create_comm = function(comm_target_name, model_id, metadata) {
+        throw new Error("Manager ._create_comm not implemented");
```

Comment:
I'm assuming you didn't mean to put a space after `Manager` and before the `.` here or below.